romancal/resample/resample_utils.py

```diff
@@ -80,7 +80,7 @@ def make_output_wcs(
     )
 
     # Check that the output data shape has no zero length dimensions
-    if not np.product(output_wcs.array_shape):
+    if not np.prod(output_wcs.array_shape):
```

Comment:
This is a tiny bit of deprecated NumPy API which was autofixed by the new rule set.